GitUI/Editor/RebaseTodoHighlightingStrategy.cs

```diff
@@ -127,7 +127,7 @@ private static bool TryHighlightInteractiveRebaseCommand(IDocument document, Lin
                         {
                             var idLength = index - idStartIndex;
 
-                            if (idLength <= 5)
+                            if (idLength < 4)
```

Comment:
This will highlight a 3 byte SHA too. Is that intentional?

Reply:
The minimum length is 4 so 3 should have been excluded. But it makes not much difference, should be changed if updating again

Reply:
This is correct, a sha with 4 chars is highlighted, 3 is not